download.html

```diff
@@ -87,14 +87,26 @@ <h2>Community Projects</h2>
       We only list community projects, use at your own risk.
     </div>
     <div class="community-content">
-        <div class="community-row">
-            <a href="https://github.com/grinfans/Niffler/releases" target="_blank"><img class="community-row-img" src="assets/images/niffler.png" /></a>
-            <div class="community-row-content">
-              <div class="community-row-heading"><a href="https://github.com/grinfans/Niffler/releases" target="_blank">Niffler</a></div>
-              <div class="community-row-desc">Grin GUI node and wallet.</div>
-              <div class="community-row-category">Windows and macOS</div>
-            </div>
-          </div>
+      <div class="community-row">
+        <a href="https://github.com/grinfans/Niffler/releases" target="_blank"><img class="community-row-img"
+            src="assets/images/niffler.png" /></a>
+        <div class="community-row-content">
+          <div class="community-row-heading"><a href="https://github.com/grinfans/Niffler/releases"
+              target="_blank">Niffler</a></div>
+          <div class="community-row-desc">Grin GUI node and wallet.</div>
+          <div class="community-row-category">Windows and macOS</div>
+        </div>
+      </div>
+      <div class="community-row">
+        <a href="https://grinplusplus.github.io" target="_blank"><img class="community-row-img"
+            src="assets/images/gpp.svg" /></a>
+        <div class="community-row-content">
+          <div class="community-row-heading"><a href="https://grinplusplus.github.io" target="_blank">
+              Grin++</a></div>
+          <div class="community-row-desc">A C++ Grin Node For Windows, macOS & Linux.</div>
```

Comment:
Let's go with `Grin GUI node and wallet written in C++.` since we already cover the OS's in the next line.

Reply:
Indeed fixed.